### PR TITLE
TRAVIS_FIXED: Fixed failing travis builds

### PR DESCRIPTION
**Pull request by corneliouzbett**
## Issue I worked on

see https://issues.openmrs.org/browse/SYNCT-

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by corneliouzbett** [comment]
The error was caused by the Ubuntu distribution used by task

**Comment by corneliouzbett** [comment]
I agree with @joeldenning that other modules do have this in their Travis config file.

This is because they use `openjdk8` but for this module, it's different, it uses `oraclejdk8` which runs on a different ubuntu distribution. I chose to maintain `oraclejdk8` and just change the environment to `trusty`.

The other fix was to change from `oraclejdk8` to `openjdk8`. 

Also added `sudo: false` for faster Travis builds

**Comment by dkayiwa** [comment]
We have been fixing this travis build by simply switching to openjdk as you can see here: https://github.com/openmrs/openmrs-module-reporting/commit/ea842ace1958df33baba4277129f51d1a07fc53d

Some more details here: https://talk.openmrs.org/t/travis-fails-to-build-project-abnomally/23165/2

**Comment by corneliouzbett** [comment]
@dkayiwa Have just updated the PR. switch from oraclejdk8 to openjdk8

**Comment by jonathandick** [comment]
Thanks @dkayiwa. Could you help us understand the process for approving and for merging prs for this module?

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 language: java
+sudo: false
 jdk:
-  - oraclejdk8
+  - openjdk8
+script: mvn clean install --batch-mode
```
Comment (dkayiwa) [inline .travis.yml]: Is the above on line 5 related to the travis failure?

**Comment by dkayiwa** [comment]
@jonathandick when two people review and approve, the pull request can be merged. If that is done, and you do not have any among the team with merge privileges, we can give the necessary permissions to the repository.

**Comment by jonathandick** [comment]
Thanks Daniel. It would be great if you provide mf squad and Cornelius Bett appropriate privileges to merge.

**Comment by dkayiwa** [comment]
@jonathandick done.

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,5 @@
 language: java
+sudo: false
```
[the remark this answers is not in the document]
Comment (corneliouzbett) [reply .travis.yml]: No
Added for faster Travis builds

File: .travis.yml (hunk 1 above)
Comment (corneliouzbett) [reply .travis.yml]: No
I added it to check for failing tests and success generating Omod

File: .travis.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dkayiwa) [reply .travis.yml]: Our convention has been for the commit message to reflect the actual changes.

File: .travis.yml (hunk 1 above)
Comment (dkayiwa) [reply .travis.yml]: Our convention has been for the commit message to reflect the actual changes.

**Pull request merged**